src/test/java/org/folio/entlinks/controller/AuthorityControllerIT.java

```diff
@@ -173,7 +207,7 @@ void getById_positive_foundByIdForExistingEntity() throws Exception {
   @Test
   @DisplayName("POST: create new Authority with defined ID")
   void createAuthority_positive_entityCreatedWithProvidedId() throws Exception {
-    assumeTrue(databaseHelper.countRows(AUTHORITY_TABLE, TENANT_ID) == 0);
+    //assumeTrue(databaseHelper.countRows(AUTHORITY_TABLE, TENANT_ID) == 0);
```

Comment:
Why it's commented?

Reply:
was just verifying some things, then forgot to uncomment